### ISSUE TITLE
possible issue with tutorial

### ISSUE THREAD

**Issue by nicholasjhorton** [ProjectMOSAIC/ggformula#62]
When I run:

learnr::run_tutorial("introduction", package = "ggformula")

I get an error message: ERROR: path[1]="": No such file or directory

> sessionInfo()
R version 3.4.1 (2017-06-30)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS High Sierra 10.13.2

Matrix products: default
BLAS: /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/3.4/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] bindrcpp_0.2      NHANES_2.1.0      mosaic_1.1.1      Matrix_1.2-12    
 [5] mosaicData_0.14.0 lattice_0.20-35   dplyr_0.7.4       ggformula_0.6    
 [9] ggplot2_2.2.1     tibble_1.3.4      learnr_0.9.1      shiny_1.0.5      
[13] RMySQL_0.10.13    DBI_0.7          

loaded via a namespace (and not attached):
 [1] reshape2_1.4.2   purrr_0.2.4      splines_3.4.1    colorspace_1.3-2 htmltools_0.3.6 
 [6] yaml_2.1.14      rlang_0.1.4      withr_2.1.0      foreign_0.8-69   glue_1.2.0      
[11] bindr_0.1        plyr_1.8.4       mosaicCore_0.4.2 stringr_1.2.0    munsell_0.4.3   
[16] gtable_0.2.0     htmlwidgets_0.9  psych_1.7.8      evaluate_0.10.1  knitr_1.17      
[21] httpuv_1.3.5     parallel_3.4.1   markdown_0.8     broom_0.4.3      Rcpp_0.12.14    
[26] xtable_1.8-2     scales_0.5.0     backports_1.1.1  jsonlite_1.5     mime_0.5        
[31] gridExtra_2.3    mnormt_1.5-5     digest_0.6.12    stringi_1.1.6    grid_3.4.1      
[36] rprojroot_1.2    tools_3.4.1      magrittr_1.5     lazyeval_0.2.1   ggdendro_0.1-20 
[41] tidyr_0.7.2      pkgconfig_2.0.1  MASS_7.3-47      assertthat_0.2.0 rmarkdown_1.8   
[46] R6_2.2.2         nlme_3.1-131     compiler_3.4.1

**Comment by rpruim** [comment]
I can confirm

```
Listening on http://127.0.0.1:6049
Warning: Error in normalizePath: path[1]="": No such file or directory
Stack trace (innermost first):
    8: <Anonymous>
    7: do.call
    6: rmarkdown::run
    5: force
    4: withr::with_envvar
    3: force
    2: withr::with_dir
    1: learnr::run_tutorial
Error : path[1]="": No such file or directory
```
Interestingly, I don't think anything has changed on the `ggformula` end of things.  So I'll have to see if this is change elsewhere.

**Comment by rpruim** [comment]
I'm making some progress, but I don't really understand the issue very well.

For the introduction tutorial, I deleted the line telling it to be progressive, then built, then put the line back in and built again.  Everything seemed to work.  The most notable this is that there is now a `figure-html4` directory that wasn't there before.  The edits seemed to be what was needed to get the HTML file rebuilt.

For the refining tutorial, none of this worked.  Even creating a new tutorial from scratch failed.  So I down-graded from a daily build to the current stable release of RStudio and tried again.  That seems to have done the trick.

I did see this https://github.com/rstudio/rmarkdown/issues/1213, which might be related.
